### DISCUSSION TITLE
Quarto - Quarto 1.4

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11420] [Blog]
# Quarto - Quarto 1.4

Quarto 1.4 brings new formats for dashboards and Typst, a new manuscript project type, a cross-reference overhaul, Shiny for Python support, and a ton of other updates.

https://quarto.org/docs/blog/posts/2024-01-24-1.4-release/

**Comment by polc1410** [comment]
The online syntax section:

Should each example actually be:

`{language} radius` rather than `{language} x`

For example

```{r}
radius <- 5
```

The radius of the circle is `{r} radius`

**Comment by cavo789** [comment]
Wonderful work!

**Reply by BB1464** [reply]
This is great; thank you very much!!!

**Comment by BB1464** [comment]
This is great; thank you very much!!!

**Comment by marianklose** [comment]
Amazing! Thanks a lot!

**Comment by Abubakar-ibrahim** [comment]
Excellent! Thank you for this update.
